docs/peering.md

```diff
@@ -195,3 +201,45 @@ Example successful response:
   }
 }
 ```
+
+### Withdraw Peer Liquidity
+
+Query:
+
+```
+mutation CreatePeerLiquidityWithdrawal ($input: CreatePeerLiquidityWithdrawalInput!) {
+  createPeerLiquidityWithdrawal(input: $input) {
+    code
+    success
+    message
+    error
+  }
+}
+```
+
+Query Variables (substitute the peer ID from the "create peer" response for `INSERT_PEER_ID`):
+
+```
+{
+  "input": {
+    "peerId": "INSERT_PEER_ID",
+    "amount": "5000",
+    "id": "25e4ae5b-e844-49f6-89fa-34f33bc03278"
+  }
+}
+```
+
+Example successful response:
+
+```
+{
+  "data": {
+    "createPeerLiquidityWithdrawal": {
+      "code": "200",
+      "success": true,
+      "message": "Created peer liquidity withdrawal",
+      "error": null
+    }
+  }
+}
+```
```

Comment:
Withdrawals are two-phase so it'll also need
https://github.com/interledger/rafiki/blob/cc2ae3f2bec056c758255d238297181bfb4acf72/packages/backend/src/graphql/schema.graphql#L83-L87

Reply:
🤭 totally forgot

Reply:
I think I found a bug. The `createPeerLiquidityWithdrawal` responds with `LiquidityMutationResponse` which only consists of 
```
type LiquidityMutationResponse implements MutationResponse {
  code: String!
  success: Boolean!
  message: String!
  error: LiquidityError
}
```
Hence, we don't get the withdrawal id to post the withdrawal.

Reply:
- #1125